mm2src/mm2_main/tests/docker_tests/mod.rs

```diff
@@ -9,7 +9,3 @@ mod swaps_confs_settings_sync_tests;
 mod swaps_file_lock_tests;
 
 #[cfg(feature = "enable-solana")] mod solana_tests;
-
-// dummy test helping IDE to recognize this as test module
-#[test]
-fn dummy() { assert!(true) }
```

Comment:
Please leave this, I am using latest clion which is a standard and one of the most used IDEs for rust and it doesn't recognize this https://github.com/KomodoPlatform/atomicDEX-API/blob/75663fff7f3f052b0b153d2695e7b8b84480513f/mm2src/mm2_main/tests/docker_tests_main.rs#L29 as a test mod unless an actual test exists here.

Reply:
I renamed the branch and had to `set-upstream` that is why I pushed different branch, sorry